knapsack-gosdk/go.mod

```diff
@@ -1,8 +1,14 @@
 module example.com/your_project/knapsack-gosdk
 
-go 1.19
+go 1.21
 
-require github.com/nextmv-io/sdk v1.2.0
+toolchain go1.22.0
+
+require (
+	github.com/nextmv-io/go-highs v0.0.0-20240304150021-94d2d68aea08
```

Comment:
I don’t think we should update community apps until we have an "official" release of these repos.

Reply:
I thought I mentioned it somewhere, but I apparently haven't. Yes, we need to cleanup the go.mods and fix the versions once we did the first release from the new location. This wasn't going to ship like this, but is as ready as it can be at this point. I'll look into the other points though, as they may be weird.